app/config/config.go

```diff
@@ -0,0 +1,28 @@
+package config
+
+import (
+	"github.com/spf13/viper"
+)
+
+var (
+	Manager *viper.Viper = viper.New()
+)
+
```

Comment:
I thought that cfg.Manager/config.Manager is explicit enough. But ConfigManager lgtm too.